pages/glossary/context.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Context
+key: context
+---
+
+The context of a piece of information is any other information that helps a user understand it by clarifying its meaning.
```

Comment:
This is extremely vague. Can you clarify this?

Reply:
Definition removed.